docs/product/components/sidebar-widgets.html

```diff
@@ -381,7 +381,7 @@
                 </div>
                 <ul class="s-sidebarwidget--content s-sidebarwidget__items">
                     <li class="s-sidebarwidget--item"><a href="#">Edit Profile</a></li>
-                    <li class="s-sidebarwidget--item" aria-current="page"><a href="#">Developer Story Preferences</a></li>
+                    <li class="s-sidebarwidget--item s-block-link s-block-link__left is-active" aria-current="page"><a href="#">Developer Story Preferences</a></li>
```

Comment:
Is this meant to be here? It's incomplete / not working as expected.

![Screen Shot 2019-04-05 at 1 10 28 PM](https://user-images.githubusercontent.com/662837/55653867-a40bca80-57a4-11e9-88d9-695985bec6d5.png)

Reply:
Good catch. I was exploring refactoring those to use `s-block-link` but will most likely punt on that.